abseil/pkg.yaml

```diff
@@ -13,6 +14,9 @@ steps:
         destination: abseil.tar.gz
         sha256: "{{ .abseil_sha256 }}"
         sha512: "{{ .abseil_sha512 }}"
+    env:
+      # FIXME: usrmerge in toolchain (?)
+      LD_LIBRARY_PATH: /usr/lib64:/usr/lib:/lib64:/lib
```

Comment:
These issues arise from usrmerge structure and should be fixable by either adjusting symlinks or search paths.